Tests/AsyncAlgorithmsTests/TestThrottle.swift

```diff
@@ -11,6 +11,7 @@
 
 import XCTest
 import AsyncAlgorithms
+import AsyncAlgorithms_XCTest
```

Comment:
I presume this is needed for the clock tests eh?

Reply:
This must have been a miss on my part, I have pushed a commit that removes this.